File: PHP PowerPoint Examples/ShapesApiDemo/saveShapeAs.php
<?php

use Spire\Cloud\Powerpoint\Sdk\Api\ShapesApi;
use Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new ShapesApi($configuration);

$name = "SlideShapes.pptx";
$password = null;
$folder = "input";
$storage = null;
$slideIndex = 0;
$shapeIndex = "1";
$format = "png"; //Available format: png, jpg, bmp
$outPath = "Shapes/SaveShapeAs.png";
$scaleX = 1.0;
$scaleY = 1.0;
$bounds = null;
$fontsFolder = null;
$apiInstance->saveShapeAs($name, $slideIndex, $shapeIndex, $format, $outPath,$password, $folder, $storage,$scaleX,$scaleY,$bounds,$fontsFolder);

File: Python PowerPoint Examples/ShapesApiDemo/saveShapeAs.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudpowerpoint.api.shapes_api.ShapesApi(configuration)

fileName = "SlideShapes.pptx"
slide_index = 0
shape_index = "1"
#supports format: Png/Jpg/Bmp/
format = "png"
dest_file_path = "Shapes/SaveShapeAs.png"
password = ""
storage = ""
scaleX = 1.0
scaleY = 1.0
bounds = 0
fonts_folder = ""
inputFolder = "input"
result =api.save_shape_as(fileName, slide_index, shape_index, format, dest_file_path,password=password, folder=inputFolder, storage=storage, scale_x=scaleX, scale_y=scaleY, bounds=bounds, fonts_folder=fonts_folder)